recipes/service_server.rb

```diff
@@ -39,7 +39,7 @@
 
 %w{ obs-source_service }.each do |package_name|
   package package_name do
-    action :install
+    action :upgrade
```

Comment:
IIRC this didn't work correctly with some Chef versions. Did you verify that this works now? Are there any new requirements on the Chef version used?